### PR TITLE
Added GeneralPhpdocAnnotationRemoveFixer, GeneralPhpdocAnnotationRenameFixer

### PR DESCRIPTION
**Pull request by keradus**
I really like to remove annotations of my choice without creating custom fixer each time

### CODE REVIEW

**Comment by SpacePossum** [comment]
Nice idea. This is missing integration tests/priority testing?

File: tests/FixerTest.php [hunk 1]
```diff
@@ -138,10 +138,17 @@ public function testFixersAreFinal(\ReflectionClass $class)
 
     public function provideFixersForFinalCheckCases()
     {
+        static $exceptions = array(
```
Comment (keradus) [inline tests/FixerTest.php]: it's from prev. implementation, should not be here.

File: src/Fixer/Phpdoc/GeneralPhpdocAnnotationRemoveFixer.php [hunk 2]
```diff
@@ -10,19 +10,45 @@
...
 use PhpCsFixer\Tokenizer\Tokens;
 
 /**
  * @author Graham Campbell <graham@mineuk.com>
- *
- * @internal
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
  */
-abstract class AbstractAnnotationRemovalFixer extends AbstractFixer
+class GeneralPhpdocAnnotationRemoveFixer extends AbstractFixer
```
Comment (SpacePossum) [inline src/Fixer/Phpdoc/GeneralPhpdocAnnotationRemoveFixer.php]: `final`

File: src/Fixer/Phpdoc/GeneralPhpdocAnnotationRenameFixer.php [hunk 3]
```diff
@@ -10,33 +10,35 @@
...
 /**
- * This abstract fixer provides a base for fixers to rename tags.
- *
  * @author Graham Campbell <graham@mineuk.com>
- *
- * @internal
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
  */
-abstract class AbstractPhpdocTagsFixer extends AbstractFixer
+class GeneralPhpdocAnnotationRenameFixer extends AbstractFixer
```
Comment (SpacePossum) [inline src/Fixer/Phpdoc/GeneralPhpdocAnnotationRenameFixer.php]: `final`

File: src/Fixer/Phpdoc/GeneralPhpdocAnnotationRemoveFixer.php (hunk 2 above)
Comment (keradus) [inline src/Fixer/Phpdoc/GeneralPhpdocAnnotationRemoveFixer.php]: duplicates https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1779#discussion_r52979856

File: src/Fixer/Phpdoc/GeneralPhpdocAnnotationRenameFixer.php (hunk 3 above)
Comment (keradus) [inline src/Fixer/Phpdoc/GeneralPhpdocAnnotationRenameFixer.php]: duplicates https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1779#discussion_r52979856

Comment (SpacePossum) [inline src/Fixer/Phpdoc/GeneralPhpdocAnnotationRenameFixer.php]: sorries about that!

**Comment by keradus** [comment]
for now test are keep for used, specific fixers

**Comment by SpacePossum** [comment]
what happend @keradus ?

**Pull request closed without merging**